packages/web-client/app/models/animated-workflow.ts

```diff
@@ -137,6 +145,13 @@ export default class AnimatedWorkflow {
 
   constructor(model: Workflow) {
     this.model = model;
+    // set this so css animations shorten their duration to at most the
+    // rate of release
+    // this is purely for devx
+    document.documentElement.style.setProperty(
+      '--thread-animation-interval',
+      `${interval}ms`
+    );
```

Comment:
love it, I haven’t been using CSS variables enough and didn’t know this was possible, seems like an excellent way to inject dynamism 🤩